src/test/java/com/fullteaching/e2e/no_elastest/functional/test/LoggedLinksTests.java

```diff
@@ -21,39 +19,42 @@
 import java.util.Map;
 import java.util.stream.Stream;
 
-import static java.lang.invoke.MethodHandles.lookup;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.slf4j.LoggerFactory.getLogger;
 
-@ExtendWith(SeleniumJupiter.class)
 class LoggedLinksTests extends BaseLoggedTest {
 
 
 
+
+
```

Comment:
There was three blank lines that should be only one, and you have added two more. This is common in this class.